vector_add_kernel.cu

```diff
@@ -0,0 +1,8 @@
+
+
```

Comment:
Remove the empty lines?
```suggestion
```

Reply:
Done in [936c038](https://github.com/nlesc-recruit/cudawrappers-usage-example-cmake-pull/pull/1/commits/936c038409d8a17ddde0a3b29f94c85b4938613c)